ctags/main/entry.c

```diff
@@ -114,6 +150,14 @@ extern const char *tagFileName (void)
 *   Pseudo tag support
 */
 
+extern void abort_if_ferror(MIO *const mio)
+{
+#ifndef CTAGS_LIB
+	if (mio_error (mio))
+		error (FATAL | PERROR, "cannot write tag file");
+#endif
```

Comment:
We never write the tag file so this should never be called. I could probably remove the ifdefs around it, it was just I was worried someone else might decide to call it in the future in ctags since the function is exported.